theme/posconvert_page.tpl.php

```diff
@@ -11,7 +11,14 @@
 
 <div id="div-page-ctr-panel">
   <div class="messages warning">
-    <strong>This is a <em>Beta Release</em> and is still undergoing testing.</strong>
+    <p>
+      <strong>This is a <em>Beta Release</em> and is still undergoing testing.</strong>
+      <em style="color: red; font-weight: bold">Important:</em>
+      Posconvert is limited to files that are less than <?php print $posconvert_max_file_limit; ?>b or <?php print number_format($posconvert_max_line_limit); ?> lines (not including the header).
+      It will take up to an hour to conver a file  of <?php print number_format($posconvert_max_line_limit); ?> lines. If you have a larger file, please
```

Comment:
"conver" should be "convert" and we can probably drop "a file of" to shorten the message a bit.